extensions/azurePublishNew/src/node/types.ts

```diff
@@ -9,6 +9,7 @@ export type ResourceProvisionService = {
   getDependencies: () => string[];
   getRecommendationForProject: (project: IBotProject) => 'required' | 'optional' | 'notAllowed';
   provision: ProvisionMethod;
+  canPollStatus: boolean;
```

Comment:
I don't understand this boolean.  I was thinking that the provision method would take the status update callback (which is what I am going to try and define next). Can you tell me more about this boolean?

Reply:
Some of the SDK create/update methods return a special `LROPoller` object with methods for interacting with long running Azure operations (response 202). I thought it would be cool to use that. 
But some (such as App Reg, Bot Channel Registration, and App Insights) won't have that - those will just take the onUploadProgress/onDownloadProgress callbacks.